tahoe_sites/models.py

```diff
@@ -0,0 +1,62 @@
+import uuid
+
+from django.conf import settings
+from django.db import models
+from organizations.models import Organization
+
+
+class UserOrganizationMapping(models.Model):
+    """
+    User membership in an organization.
+
+    Tahoe's fundamental multi-site relationship.
+    """
+
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    organization = models.ForeignKey(Organization, on_delete=models.CASCADE)
+    is_active = models.BooleanField(default=True)
+    is_amc_admin = models.BooleanField(default=False)
```

Comment:
`is_organization_admin` or `is_admin` are more clear since this is an LMS context and not AMC context.

```suggestion
    is_admin = models.BooleanField(
        default=False,
        # TODO: Remove once we migrate off edx-organizations tables
        db_field='is_amc_admin' if settings.FEATURES['TAHOE_SITES_USE_ORGS_MODELS'] else None,
    )
```

Also, AMC is going to be replaced by Dashboard 2.0.

Reply:
done, with a change